TombEngine/Scripting/Internal/TEN/Flow/FlowHandler.cpp

```diff
@@ -425,6 +426,13 @@ bool FlowHandler::DoFlow()
 		}
 		else
 		{
+			// Check if there is at least 1 playable level
+			if (g_GameFlow->GetNumLevels() == 1)
```

Comment:
I am not sure TEN should close if there is no levels exist on the script but title does. Technically, this is correct scenario, and "crashing" engine in such scenario more resembles TRNG or original engine behaviour. I'd just throw a warning and allow engine to load and play title level anyway.